bspump/avro/deserializer.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import logging
+import io
+import bspump.avro
+import fastavro
+import bspump.trigger
```

Comment:
There you  are in the bspump, so you need to use relative imports (bspump.avro, bspump.trigger)